modules/installation-creating-aws-worker.adoc

```diff
@@ -180,9 +180,9 @@ the CloudFormation template for the security group and roles.
 section of this topic and save it as a YAML file on your computer. This template
 describes the networking objects and load balancers that your cluster requires.
 
-. If you specified an `m5` instance type as the value for `WorkerInstanceType`,
-add that instance type to the `WorkerInstanceType.AllowedValues` parameter
-in the CloudFormation template.
+. Optional: If you specified an `m5` instance type as the value for `WorkerInstanceType`, add that instance type to the `WorkerInstanceType.AllowedValues` parameter in the CloudFormation template.
```

Comment:
All the allowed values  had been removed by https://github.com/openshift/installer/commit/5db41b0a5055c1a98c92f8899c4689ec75cf4bc8, I think we can remove this `Optional` part.